browse-scaler/src/cache.js

```diff
@@ -21,8 +21,11 @@ exports.cacheImage = (key, image) => {
 
 exports.getImageFromCache = async collectionId => {
   try {
+    console.log("attempting to get image from cache");
     const image = await getAsync(collectionId);
-    return image;
+
+    console.log(`got image from cache ${image}`);
```

Comment:
Three lines above you use double quotes. There seems to be a bunch of changes in this file between single and double quotes.

What's the standard? Is it different if you need to do variable substitution?

Reply:
These are backticks which are used for string substitutions

Reply:
We should stick to single quotes unless doing interpolation (template literals), which uses back ticks. Single quotes are pretty much the standard now.

Reply:
The air-bnb style guide (which we are using) says use single quotes (unless doing interpolation, which requires backticks). The linter should be enforcing this. Have you run the linter on this project?

Reply:
I somehow didn't copy the eslint and prettier config files, so I fixed that and am linting properly again